cpmpy/expressions/python_builtins.py

```diff
@@ -19,9 +19,11 @@
         sum
 """
 import numpy as np
-from .variables import NDVarArray
+from fishhook import *
```

Comment:
this would require updating the requirements file then...


I would like all the fishooks to be in a new file 'cpmpy/expressions/fishooks..py' or similar. Also ideally the code would also work if somebody does not have fishook installed? or maybe not... but then definitely update requirements.

Reply:
Yes we should update the requirements